docs/data/base/components/textarea-autosize/textarea-autosize.md

```diff
@@ -6,24 +6,40 @@ githubLabel: 'component: TextareaAutosize'
 packageName: '@mui/base'
 ---
 
-# Textarea Autosize
+# Textarea autosize
```

Comment:
This makes me think, should we change the title above to something like: "title: Unsyled Textarea Autosize React component" similar to the other unstyled components?

Reply:
Good question! I left it off since it's not in the component name. But I could go either way with it.